examples/quickstart/jupyter-notebooks/nested-columns-tutorial.ipynb

```diff
@@ -0,0 +1,517 @@
+    "\n",
+    "print(json.dumps(json.loads(response.text), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "240b0ad5-48f2-4737-b12b-5fd5f98da300",
+   "metadata": {},
+   "source": [
+    "## Transform nested data\n",
+    "\n",
+    "You can use Druid's [SQL JSON functions](https://druid.apache.org/docs/latest/querying/sql-json-functions.html) to transform nested data in your ingestion query.\n",
+    "\n",
+    "Run the following cell to insert sample data into a new datasource named `kttm_transform`. The SELECT query extracts the `country` and `city` elements from the nested `geo_ip` column and creates a composite object `sessionDetails` containing  `session` and `session_length`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1f21fa8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql/task\"\n",
+    "print(druid_host+endpoint)\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "\"query\": \"INSERT INTO \\\"kttm_transform\\\" \\\n",
+    "    WITH \\\"source\\\" AS \\\n",
+    "    (SELECT * FROM TABLE(EXTERN('{\\\"type\\\":\\\"http\\\",\\\"uris\\\":[\\\"https://static.imply.io/example-data/kttm-nested-v2/kttm-nested-v2-2019-08-25.json.gz\\\"]}', \\\n",
+    "       '{\\\"type\\\":\\\"json\\\"}','[{\\\"name\\\":\\\"timestamp\\\",\\\"type\\\":\\\"string\\\"},{\\\"name\\\":\\\"session\\\",\\\"type\\\":\\\"string\\\"},{\\\"name\\\":\\\"session_length\\\",\\\"type\\\":\\\"string\\\"}, \\\n",
+    "        {\\\"name\\\":\\\"event\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"},{\\\"name\\\":\\\"agent\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"},{\\\"name\\\":\\\"geo_ip\\\",\\\"type\\\":\\\"COMPLEX<json>\\\"}]'))) \\\n",
+    "        SELECT TIME_PARSE(\\\"timestamp\\\") AS \\\"__time\\\", \\\n",
+    "        JSON_QUERY(geo_ip, '$.country') as country, \\\n",
+    "        JSON_QUERY(geo_ip, '$.city') as city, \\\n",
+    "        JSON_OBJECT('session':session, 'session_length':session_length) as sessionDetails \\\n",
+    "        FROM \\\"source\\\" \\\n",
+    "    PARTITIONED BY DAY\"\n",
+    "})\n",
+    "\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "ingestion_taskId_response = response\n",
+    "ingestion_taskId = json.loads(ingestion_taskId_response.text)['taskId']\n",
+    "\n",
+    "print(f\"\\nInserting data into the table named {dataSource2}\")\n",
+    "print(\"\\nThe response includes the task ID and the status: \" + response.text + \".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb22bd78",
+   "metadata": {},
+   "source": [
+    "When the ingestion task is complete, run the following cell to query the data and return `country`, `city`, and `sessionDetails` from 3 rows:"
```

Comment:
You probably want to include a similar while + sleep loop like from above so that they get feedback for when it's complete. Without it, there's no response/indicaiton for when it's done unless they manually run an API call themselves  or if they're monitoring it in the Druid console

Reply:
Updated.